metadata-suggestions.md

```diff
@@ -0,0 +1,682 @@
+          },
+         "differences": {
+           "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValueDifferences/0000-0003-3681-2038"
+         }
+        }
+      }
+    ]
+  }
+}
+```
+
+The differences are explained below, and won't be embedded in this endpoint.
+
+### Single entry
+**/api/integration/metadata-suggestions/<:suggestion-name>/entryValues/<:entry-id>**  
+
+Parameters:
+* workspaceitem: the current workspace item ID (mutually exclusive with workflowitem)
+* workflowitem: the current workflow item ID (mutually exclusive with workspaceitem)
+
+It returns the data from one entry in a live import source
+
+sample for a live import source /api/integration/metadata-suggestions/orcid/entryValues/0000-0002-4271-0436
+```json
+{
+  "id": "0000-0002-4271-0436",
+  "display": "Smith, Dean",
+  "value": "Smith, Dean",
+  "type": "metadataSuggestionEntry",
+  "metadataSuggestion": "orcid",
+  "metadata": {
+    "dc.identifier.orcid": [
+      {
+        "value": "0000-0002-4271-0436",
+        "language": null,
+        "authority": null,
+        "confidence": 0,
+        "place": -1
+      }
+    ],
+    "dc.identifier.uri": [
+      {
+        "value": "https://orcid.org/0000-0002-4271-0436",
+        "language": null,
+        "authority": null,
+        "confidence": 0,
+        "place": -1
+      }
+    ],
+    "person.familyName": [
+      {
+        "value": "Smith",
+        "language": null,
+        "authority": null,
+        "confidence": 0,
+        "place": -1
+      }
+    ],
+    "person.givenName": [
+      {
+        "value": "Dean",
+        "language": null,
+        "authority": null,
+        "confidence": 0,
+        "place": -1
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValues/0000-0002-4271-0436"
+    },
+    "differences": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValueDifferences/0000-0002-4271-0436"
+    }
+  },
+  "_embedded" : {
+     "differences" : {
+         "dc.identifier.orcid" : {
+           "currentvalues" : [],
+           "suggestions" : [
+             {
+               "operations": [ "add/metadata/dc.identifier.orcid/-" ],
+               "newvalue": "0000-0002-4271-0436"
+             }
+           ]
+         },
+         "dc.identifier.uri" : {
+           "currentvalues" : [ "https://profiles.utsouthwestern.edu/profile/16780/dean-smith.html" ],
+           "suggestions" : [
+             {
+               "operations": [ "add/metadata/dc.identifier.uri/-", "replace/metadata/dc.identifier.uri/0" ],
+               "newvalue": "https://orcid.org/0000-0002-4271-0436"
+             }
+           ]
+         }
+     }
+  }
+}
+```
+
+The differences embedded are the suggested metadata changes to be applied to the given item
```

Comment:
I'm not sure we should embed the differences here, as that's a separate endpoint now.  I'm fine with having the `_links` list the `differences` endpoint though.  I just think the differences should be a separate REST call, as it's a separate endpoint.

Reply:
I've removed the embedding since the removal of the parameters you suggested at https://github.com/DSpace/Rest7Contract/pull/83#discussion_r354522395 implies the differences can't be embedded anymore